### ISSUE TITLE
Accessibility in new pagination component version

### ISSUE THREAD

**Issue by eschweitzer78** [digitalnsw/nsw-design-system#302]
Hi Lauren and team

Just had a look at the new version of the pagination component. The addition of the disabled css class and illustration is very helpful and all the other updates are superb.

Wouldn't the now visually disabled anchor (with no pointer events) create an accessibility issue if it's missing an `aria-disabled` statement though?

`<a class="nsw-icon-button{{#if disabled}} disabled{{/if}}" href="#" {{#if disabled}}aria-disabled="true"{{/if}}>`

Hopefully this makes sense... You could even consider applying the right style by leveraging just the `aria-disabled` attribute as a selector. 

Thank you

**Comment by laurenhitchon** [comment]
Hi @eschweitzer78, 

Thank you for taking the time to review the updated pagination component. I'm glad to hear that you find the changes helpful.

You're absolutely right that the addition of the visually disabled anchor (with no pointer events) could create an accessibility issue as the anchor is missing an aria-disabled statement. I have added the aria-disabled="true" attribute to the anchor element (using your suggested code) to indicate that it is currently disabled and cannot be interacted with.

Thanks again for your feedback and let us know if you have any other suggestions.
